### PR TITLE
Adding an event stream

### PR DESCRIPTION
**Pull request by zunder**
Do not merge

### CODE REVIEW

File: src/main/scala/dcos/metronome/TestClient.scala [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    response.entity.dataBytes.runWith(printer)
+  }
+
+  def subscribe(): Future[Done] = {
+    val responseSink = Sink.foreach[HttpResponse] { response =>
+      response.entity.dataBytes.map(toEvent).runWith(printResponseSink)
+    }
+    request(call, responseSink)
+  }
+*/
```
Comment (janisz) [inline src/main/scala/dcos/metronome/TestClient.scala]: Why this is commented out?

File: src/main/scala/dcos/metronome/TestClient.scala [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      |}
+      |
+    """.stripMargin
+
+  val data = Json.parse(json)
+
+  val bytesReturned: Future[Done] = eventStreamResponse.flatMap {
+    res =>
+      val sink = Sink.foreach[ByteString] { bytes =>
+        Thread.sleep(500000)
```
Comment (janisz) [inline src/main/scala/dcos/metronome/TestClient.scala]: Why do we need to sleep?

File: api/src/main/resources/routes [hunk 3]
```diff
@@ -16,5 +15,5 @@ GET /job-runs dcos.metronome.api.v1.controllers.JobRunController.getAllJobRuns
 GET /job-runs/*id dcos.metronome.api.v1.controllers.JobRunController.getJobRun(id)
 POST /job-runs/*id dcos.metronome.api.v1.controllers.JobRunController.changeJobRun(id, action)
 
-
+GET /events dcos.metronome.api.v1.controllers.EventStreamController.connect
```
Comment (janisz) [inline api/src/main/resources/routes]: Update the docs as well

File: api/src/main/scala/dcos/metronome/api/v1/models/package.scala [hunk 4]
```diff
@@ -20,7 +21,7 @@ package object models {
   implicit val errorFormat: Format[ErrorDetail] = Json.format[ErrorDetail]
   implicit val unknownJobsFormat: Format[UnknownJob] = Json.format[UnknownJob]
 
-  implicit val DateTimeFormat: Format[DateTime] = Format (
+  implicit val DateTimeFormat: Format[DateTime] = Format(
```
Comment (janisz) [inline api/src/main/scala/dcos/metronome/api/v1/models/package.scala]: Change formatting in separated commit/pr

**Comment by bydga** [comment]
Hi, is there any progress on the event stream? Why shouldnt this be merged?

**Comment by aquamatthias** [comment]
We need to rethink this better. Close for now.

**Pull request closed without merging**